### PR TITLE
Updates for SymfonyCon talk

### PR DESCRIPTION
**Pull request by willdurand**
Hi!

Here is a Work In Progress for the talk  Lukas and I will give at SymfonyCon.

William

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -198,8 +198,6 @@ It comes pre-configured with the following bundles:
 
   * [**FOSRestBundle**][16] - Adds rest functionality
 
-  * [**FSCHateoasBundle**][18] - Adds HATEOAS support
```
Comment (lsmith77) [inline README.md:201]: why remove instead of replace with BazingaHateoasBundle ?

Comment (willdurand) [inline README.md:201]: Forgot to update the doc while adding the BazingaHateoasBundle..

Comment (lsmith77) [inline README.md:201]: can you fix that .. i will then merge

Comment (willdurand) [inline README.md:201]: done.

it is up to you, but you could keep this PR open until I push everything.

Comment (lsmith77) [inline README.md:201]: ok .. lets keep it open ..

**Comment by willdurand** [comment]
#### http://localhost:8000/notes/0.json

``` json
{
    "message": "Note 0",
    "_links": {
        "self": {
            "href": "http://localhost:8000/notes/0"
        }
    }
}
```

#### http://localhost:8000/notes/1.json

``` json
{
    "message": "Note 1",
    "_links": {
        "self": {
            "href": "http://localhost:8000/notes/1"
        }
    },
    "_embedded": {
        "associated_events": [
            {
                "name": "SymfonyCon",
                "date": "2013-12-12T00:00:00+0100"
            },
            {
                "name": "Christmas Day",
                "date": "2013-12-25T00:00:00+0100"
            }
        ]
    }
}
```

**Comment by willdurand** [comment]
#### http://localhost:8000/notes.json

``` json
{
    "notes": [
        {
            "secret": "6r3Fe3c893oDXVGVkWhQ8RbIjZcyn410/MF3+skHRHgzIcYalQx00h+aqf4LBeTH6ASqNBHsW275fWv4YG1u0g==",
            "message": "Note 0",
            "_links": {
                "self": {
                    "href": "http://localhost:8000/notes/0"
                }
            }
        },
        {
            "secret": "fmRCVeaSWpSEXUNdfaCUrzvm02MvpGaHHNJv5O5ekcGkMEL470hANDBCWWqHVpn44TIq0EBzv548tLW6mOv8vA==",
            "message": "Note 1",
            "_links": {
                "self": {
                    "href": "http://localhost:8000/notes/1"
                }
            },
            "_embedded": {
                "associated_events": [
                    {
                        "name": "SymfonyCon",
                        "date": "2013-12-12T00:00:00+0100"
                    },
                    {
                        "name": "Christmas Day",
                        "date": "2013-12-25T00:00:00+0100"
                    }
                ]
            }
        }
    ],
    "limit": 5,
    "_links": {
        "self": {
            "href": "http://localhost:8000/notes"
        },
        "note": {
            "href": "http://localhost:8000/notes/{id}",
            "templated": true
        }
    }
}
```

**Comment by willdurand** [comment]
@lsmith77 WDYT?

**Comment by lsmith77** [comment]
looks good .. no need to get fancier.

**Comment by willdurand** [comment]
Hopefully, Hautelook guys will have time to look at my PRs:
- https://github.com/hautelook/TemplatedUriBundle/pull/10
- https://github.com/hautelook/TemplatedUriRouter/pull/3

**Comment by willdurand** [comment]
I will create another PR for Hautelook stuff, and maybe try to add BazingaRestExtraBundle, not sure how yet.

**Comment by lsmith77** [comment]
cool

File: src/Acme/DemoBundle/Model/Note.php [hunk 2]
```diff
@@ -24,7 +24,21 @@ class Note
...
      */
-    public function __toString() {
+    public function __toString()
+    {
         return $this->message;
     }
-}
\ No newline at end of file
+
+    public function getAssociatedEventsRel()
```
Comment (adrienbrault) [inline src/Acme/DemoBundle/Model/Note.php:32]: Is that just to highlight the feature ?

Comment (willdurand) [inline src/Acme/DemoBundle/Model/Note.php:32]: Yep

**Pull request merged**